BENEFITS.md

```diff
@@ -0,0 +1,65 @@
+# Benefits of the AdEx Network protocol for decentralized advertising 
+
+We began building AdEx Network as an alternative to the array of problems faced by the advertising industry today. 
+
+As global advertising and marketing investments continue to grow exponentially around the world, we see a significant shift in the importance carried by different advertising mediums: while TV has always until now been king for advertisers, it is rapidly being outrun by online & digital channels. Experts forecast that search and social advertising will be accountable for about 67% of ad spend growth by 2020 ([source](https://www.zenithmedia.com/insights/global-intelligence-issue-07-2018/search-and-social-to-drive-67-of-adspend-growth-by-2020/)). These two mediums are run by a mere four players: Facebook, Twitter, Google and Oath. In recent months, all of these large ad property providers (and [Facebook in particular](https://www.nytimes.com/2018/04/18/business/media/facebook-advertisers-privacy-data.html)) have faced criticism in regard to doctored reporting and selling user data to their partners - and with good reason. 
```

Comment:
our statement must be clear that we're competing to everyone *but* the big ones atm 
so perhaps we should focus less on the big players and make that clear